tcf_website/templates/course/course.html

```diff
@@ -2,42 +2,64 @@
 {% load static %}
 
 {% block styles %}
-<link rel="stylesheet" href="{% static 'course/course.css' %}" />
-<link rel="stylesheet" href="{% static 'common/rating_card.css' %}" />
-<link rel="stylesheet" href="{% static 'common/show-hide.css' %}" />
-<link rel="stylesheet" href="{% static 'common/toolbar.css' %}" />
+    <link rel="stylesheet" href="{% static 'course/course.css' %}"/>
+    <link rel="stylesheet" href="{% static 'common/rating_card.css' %}"/>
+    <link rel="stylesheet" href="{% static 'common/show-hide.css' %}"/>
+    <link rel="stylesheet" href="{% static 'common/toolbar.css' %}"/>
+    <link rel="stylesheet" href="{% static 'base/tcf-style.css' %}"/>
```

Comment:
oh you don't need to add tcf-style.css to this template - it's already imported in base (and course.html extends from base). I'd remove it!